web-console/future-openshift/cron-jobs/cron-jobs.md

```diff
@@ -0,0 +1,27 @@
+# Cron Jobs
+
+## Create Cron Job
+![](img/cron-job-list.png)
+- Clicking the `Create Cron Job` button launches the creation form.
+
+![](img/create-cron-job.png)
```

Comment:
what happens when clicking "what's this"?  Is it a link to online doc or does it provide some on screen help for the expression syntax?

Reply:
https://en.wikipedia.org/wiki/Cron is where the OpenShift docs link to about cron format, I expect we could do the same

Reply:
Do we do this anywhere else this way? Wondering if it belongs in a field level help icon or one of the other new help patterns.

Reply:
I like this :+1: